gbdxtools/images/meta.py

```diff
@@ -460,6 +461,17 @@ def _rgb_bands(self):
     def _ndvi_bands(self):
         return [6, 4]
 
+    def base_layer_match(self, **kwargs):
+        rgb = self.rgb(**kwargs)
+        if "blm" in kwargs and not kwargs["blm"]:
+            return rgb
+        from gbdxtools.images.tms_image import TmsImage
```

Comment:
move to top?

Reply:
cant move it to the top since it creates an import conflict since TmsImage relies on BaseImage/Geoimage etc